### PR TITLE
Improvments

### PR DESCRIPTION
**Pull request by Niktux**

### CODE REVIEW

**Comment by Niktux** [comment]
```make install```  is replaced by ```make init``` (or ```make``` without any args)
because of one my change : I spread cli args to ```composer``` (and ```phpunit```) target. It allows to use all composer commands without declaring one make target for each every command. But when you type ```make composer install```, make executes ```composer install``` and then the ```install``` target :-(

That's why ```install``` target was renamed into ```init``` (and set as default target)

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -8,15 +8,15 @@ services:
...
-    db:
-        image: mariadb
-        container_name: onyx-database
-        volumes:
-            - ../var/data:/var/lib/mysql
-        environment:
-            - MYSQL_ROOT_PASSWORD=root
+#    db:
+#        image: mariadb
+#        container_name: onyx-database
```
Comment (lebris) [inline docker/docker-compose.yml]: `container_name` à éviter, laisser docker gérer le nom du container `<directory>_<service name>_<index>` 

Ou préférer utiliser `COMPOSE_PROJECT_NAME` : `<COMPOSE_PROJECT_NAME>_<service name>_<index>` 

cf https://docs.docker.com/compose/reference/envvars/#/composeprojectname

Comment (Niktux) [inline docker/docker-compose.yml]: Ok

Je te laisse corriger et merger ?

Comment (lebris) [inline docker/docker-compose.yml]: yes, cf #11

**Pull request merged**